### PR TITLE
feat: Support `mll-lab/php-utils` 2

### PR DESCRIPTION
**Pull request by simbig**
- ~[ ] Added automated tests~
- ~[ ] Documented for all relevant versions~
- [x] Updated the changelog

**Changes**

- Support `mll-lab/php-utils` 2

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "require": {
     "php": "^8.1",
     "illuminate/support": "^9.51 || ^10 || ^11",
-    "mll-lab/php-utils": "^1.13",
+    "mll-lab/php-utils": "^1.13 || ^2",
```
Comment (kilya11) [inline composer.json:24]:
```suggestion
    "mll-lab/php-utils": "^2",
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,16 @@ See [GitHub releases](https://github.com/mll-lab/laravel-utils/releases).
...
 
+## v6.0.0
+
+### Added
+
+- Support `mll-lab/php-utils` 2
+
+### Changed
+
+- Drop support for `mll-lab/php-utils` ^1.3
```
Comment (spawnia) [inline CHANGELOG.md]: Why? Seems like an unnecessary breaking change that causes churn.

Comment (simbig) [reply CHANGELOG.md]: @kilya11 requested this change to keep the dependencies simple. 
I am actualy ok with both versions because both have pros and cons

**Review by kilya11 — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (kilya11) [reply CHANGELOG.md]: Sure, the simplicity has its cost. Let the [#1 contributor](https://github.com/mll-lab/laravel-utils/graphs/contributors) approve.

**Pull request merged**